README.md

```diff
@@ -245,12 +258,6 @@ Injector:
         'hello': 'Something'
```

Comment:
Just above at line 255: This needs to be changed from `RequestCache` to `RequestCacheMiddleware`

Do you know how you can comment on lines not within the change set?

Reply:
> Do you know how you can comment on lines not within the change set?

No idea 😅